### PR TITLE
Voting UI

### PR DESCRIPTION
**Pull request by robdmoore**
Fixed the merge conflict in #181. Closes #181.

### CODE REVIEW

File: components/Voting/sessionPanel.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  paddingTop: '10px',
+}))
+
+// export const Icon = styled('span')(({ theme }) => ({
+
+// }))
+
+// export const Icon = styled('span')(({ theme }) => ({
+
+// }))
```
Comment (mzaatar) [inline components/Voting/sessionPanel.styled.tsx]: I think this was left behind by mistake, wasn't it?

File: components/Voting/sessionPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+export const SessionPanel: React.FC<SessionPanelProps> = ({
+  session: s,
+  anonymousVoting,
+  isVotedFor,
+  isVotingDisabled,
+  hideVotingButtons,
+  isInShortlist,
+  expandAll,
+  toggleVote,
+  toggleShortlist,
```
Comment (mzaatar) [inline components/Voting/sessionPanel.tsx]: Minor: extra colon

Comment (robdmoore) [reply components/Voting/sessionPanel.tsx]: The prettier config deliberately puts them in there

Comment (antonjb) [reply components/Voting/sessionPanel.tsx]: Comma dangle is the best 😛

**Pull request merged**